frontend/lib/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -64,6 +81,92 @@ export const DeckGlJsonChart: FC<PropsWithHeight> = props => {
     setIsInitialized(true)
   }, [])
 
+  const handleClick = useCallback(
+    (
+      info: PickingInfo,
+      event: { srcEvent: MouseEvent | TouchEvent | PointerEvent }
+    ) => {
+      const {
+        color,
+        index,
+        picked,
+        x,
+        y,
+        pixel,
+        coordinate,
+        devicePixel,
+        pixelRatio,
+        object,
+      } = info
+
+      const layerId = `${info.layer?.id || null}`
```

Comment:
I just tried this and if a pydeck layer is created with `id=""`, pydeck assigns it its own id. Since we're already planning on mentioning that layer IDs are required for this (along with `pickable=True`), this differentiation here does not change the expected behavior of the selection. I appreciate you pointing out this edge case!

Reply:
thanks for verifying!